README.md

```diff
@@ -1,42 +1,139 @@
-# my-package-name
+# openapi-client
```

Comment:
Update metadata or configuration so that the package name is something like `stitch-connect-client` and we can do something like `import stitch_connect_client`